ui/README.md

```diff
@@ -46,7 +46,7 @@ Open the _src/Env.elm_ file and edit the values of the `env` record.
 
 Follow the instructions above to install and run GUI as a native build. In
 _src/Env.elm_ change a `url` field value of the `elm` record to
-`http://localhost:80`. Instead of `make run` you can install `elm-live`
+`http://localhost:80/`. Instead of `make run` you can install `elm-live`
```

Comment:
why that?

Reply:
UI doesn't work without trailing slash in this variable. I found this error in README while working on this issue but it's not related to pagination. Should I make seperate NOISSUE PR for this?

Reply:
Nope, here is OK. Just that this must be noted somewhere.

Reply:
ok, I resolve this one then

Reply:
Make a special note that trailing `/` is mandatory, as it is often source of errors.

> You can use markdown quote like this for this kind of notices.